src/address.js

```diff
@@ -204,10 +232,11 @@ Returns null if no address is found.
   /**
    * Retrieves the latest balance data of the given address. Returns null if no address is found.
    *
-   * @param hash - The address of the account.
-   * @param filterOptions - The filter options associated with the request.
-   * @returns The balance data of the account or if no address is found.
+   * @param {string} hash - The address of the account.
+   * @param {object} [filterOptions] - The filter options associated with the request. See [docs](https://docs.amberdata.io/reference#get-current-account-balance) for more details.
+   * @returns {Promise<object>} The balance data of the account or if no address is found.
    * @example
+   * const latestBalance = await web3data.address.getLatestBalance('0x06012c8cf97bead5deae237070f9587f8e7a266d')
    */
   getLatestBalance(hash, filterOptions = {}) {
```

Comment:
@taylorjdawson this method still is weird... We have getBalance (which i thought was latest), getLatestBalance, getHistoricalBalance. How do i know which to use in my project?

Reply:
As it stand `getBalance` get's historical or latest depending on filters.  `getLatestBalance` & `getHistoricalBalance` are the individual commands that `getBalance` uses under the hood. Not saying this is the right or preferred way but that's what it's doing.